tensorflow/compiler/xla/service/gpu/horizontal_loop_fusion.h

```diff
@@ -93,6 +93,8 @@ namespace gpu {
 // output dims of the concatenate will be used as the kernel launch dims.
 // Instruction bitcasts can be used for Reshape2 and Reshape3 as long as the
 // outputs of Mul and Add are row-major.
+//
+// Note, reshapes are added only if the tenors isn't already a vector.
```

Comment:
Super nit: "tenors" is a typo?

(I hope this isn't blocking. It's not important unless you happen to see it at a convenient time.)